manifests/cli.pp

```diff
@@ -22,9 +22,10 @@
         -> Anchor['jenkins::end']
   }
 
-  $jar = "${jenkins::libdir}/jenkins-cli.jar"
-  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/jenkins-cli.jar"
-  $move_jar = "mv WEB-INF/jenkins-cli.jar ${jar}"
+
+  $jar = "${jenkins::libdir}/cli-${::jenkins_version}.jar"
+  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/lib/cli-${jenkins_version}.jar"
```

Comment:
Wouldn't it be better to list the files in the war and grep the exact file name rather than relying on a fact?

Reply:
we can unzip and grep yes, will do

Reply:
@ekohl check the latest commit, that should work.
```
root@lga-cmpjenkins01 anesterova # jar -xf /usr/lib/jenkins/jenkins.war WEB-INF/lib/

root@lga-cmpjenkins01 anesterova # ll

drwxr-xr-x 3 root root   16 May 27 14:52 WEB-INF

root@lga-cmpjenkins01 anesterova # mv WEB-INF/lib/cli-*.jar /tmp/cli.jar

root@lga-cmpjenkins01 anesterova # ll /tmp/cli.jar

-rw-r--r-- 1 root root 3139801 May 18 14:50 /tmp/cli.jar

root@lga-cmpjenkins01 anesterova # ```